### ISSUE TITLE
"Verbose" mode

### ISSUE THREAD

**Issue by sgherbst** [zeroasiccorp/switchboard#133]
Would be nice to have a simple mechanism to print out all packets being sent, that can easily be enabled/disabled.  Alternatively, traffic could be written to a file compatible with an existing network traffic viewer.

**Comment by sgherbst** [comment]
This might be good to add after #212 is merged, since that will provide an abstraction where this sort of rerouting could be performed
